### PR TITLE
Script feature

### PR DESCRIPTION
**Pull request by leotrj**
Read() added to SensorCenter()
GPIOSensor can be created (has read() and readAll())
Register is currently commented out so that it cant add new sensors

### CODE REVIEW

File: python-interface/src/received/line_follower.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+center = Sensor(bot, 'center')
+# redundant? bot.register_sensor(left)
+# redundant? bot.register_sensor(right)
+# redundant? bot.register_sensor(center)
+
+end = False
+while(not end):
+
+    while(center.read()==1):
+        bot.move_foward(100)
```
Comment (leotrj) [inline python-interface/src/received/line_follower.py]: don't worry about it

Comment (jchen236) [inline python-interface/src/received/line_follower.py]: i did already i thinks?

File: src/main/java/simulator/simbot/ColorIntensitySensor.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        }
+        int rgb = img.getRGB(transformed[0], transformed[1]);
+        if (rgb <= -16777216) {
+            rgb=1;
+        } else {
+            rgb=0;
+        }
+        jo.addProperty("data",rgb);
+
+//        System.out.printf("%d, %d\n",transformed[0], transformed[1]);
```
Comment (jchen236) [inline src/main/java/simulator/simbot/ColorIntensitySensor.java]: Resolved

File: src/main/java/simulator/simbot/ColorIntensitySensor.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        }
+
+        VisionCoordinate parentCoordinate = allLocs.get(0).coord;
+
+        if (parentCoordinate == null) {
+            jo.addProperty("data",-1);
+            return jo;
+        }
+
+        // TODO: Coordinate transform, for now just read at the middle
```
Comment (jchen236) [inline src/main/java/simulator/simbot/ColorIntensitySensor.java]: Marking as resolved

**Pull request merged**